lib/blockchain_double.js

```diff
@@ -17,13 +17,29 @@ var estimateGas = require("./utils/gas/estimateGas");
 var _ = require("lodash");
 var promisify = require("util").promisify;
 const BN = utils.BN;
+var path = require("path");
+var targz = require("targz");
 
 function BlockchainDouble(options) {
   var self = this;
   EventEmitter.apply(self);
 
   this.options = options = this._applyDefaultOptions(options || {});
 
+  function decompressChain(tarPath, copyChainPath) {
```

Comment:
Is there a way to make ganache core wait until decompressChain has finished with decompressing the tar before proceeding? If I add a promise on the decompression and await it (like in ganache cli), the rest of the blockchain_double continues executing, even though tar decompression hasn't finished.